evaluation/check_predict.py

```diff
@@ -1,4 +1,5 @@
-#shows how to predict using the model and how it works
+# Shows how to predict using the model and how it works
+import argparse
 import numpy as np
 import sys
 sys.path.append("..")
```

Comment:
This script assumes to be run from `evaluation` directory. Please mention that in the usage string.